src/encoded/commands/generate_ontology.py

```diff
@@ -213,7 +213,8 @@
     "OBI:0002675": "MPRA",
     "OBI:0002571": "polyA plus RNA-seq",
     "OBI:0002572": "polyA minus RNA-seq",
-    "OBI:0002631": "scRNA-seq"
+    "OBI:0002631": "scRNA-seq",
+    "OBI:0002112": "small RNA-seq"
 }
```

Comment:
I think that means you need to generate a new ontology to take effect?

Reply:
I went through the [directions here ](https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/docs/updating_ontologies.md)as per Bonita's direction